lib/mirage/impl/mirage_impl_git.ml

```diff
@@ -34,7 +34,7 @@ let git_happy_eyeballs =
 
 let git_tcp =
   let packages =
-    [ package "git-mirage" ~sublibs:[ "tcp" ] ~min:"3.10.0" ~max:"3.11.0" ]
+    [ package "git-mirage" ~sublibs:[ "tcp" ] ~min:"3.11.0" ~max:"3.12.0" ]
```

Comment:
```suggestion
    [ package "git-mirage" ~sublibs:[ "tcp" ] ~min:"3.10.0" ~max:"3.12.0" ]
```

since the 3.10 series provides the same, it's fine to widen the bounds.